tests/rules/prefer-single-boolean-return.test.ts

```diff
@@ -248,7 +248,7 @@ function foo() {
               output: `
 function foo() {
   if (bar()) {
-    return baz();
+    return !!(baz());
```

Comment:
I think we said we want 2 suggestions when it's not comparison or negation expression: `return !!(baz())` and `return baz()`